security/aws/unencrypted_s3_buckets/README.md

```diff
@@ -0,0 +1,69 @@
+## AWS Unencrypted S3 Buckets
+ 
+### What it does
+This policy checks all S3 buckets in the AWS account and reports on any that do not have default encryption set. When an unencrypted S3 bucket is detected, the user can choose to enable default encryption (AES-256) after approval and the user can also choose to delete the unencrypted S3 Buckets by enabling 'delete action' as mentioned in Enable delete action section below after approval.
+ 
+### Functional Details
+ 
+The policy leverages the AWS API to determine the encryption settings for each S3 bucket.
+ 
+When an unencrypted bucket is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have an option to modify configuration after manual approval, and even Users can perform delete action if required.
+
+* encrypt - modifies the configuration of the unencrypted S3 Bucket by enabling the Default encryption i.e. AES-256
+ 
+### Input Parameters
+ 
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
+- *Ignore tags* - S3 Buckets with any of these tags will be ignored 
+ 
+### Required RightScale Roles
+ 
+- policy_manager
```

Comment:
We no longer need to have policy_manager here.

Reply:
I had taken care this, thanks.